Source/SelfService/Web/App.tsx

```diff
@@ -41,18 +41,15 @@ import { ContainerRegistryScreen } from './screens/containerRegistryScreen';
 import { M3ConnectorScreen } from './screens/m3connectorScreen';
 import { LogsScreen } from './screens/logsScreen';
 
-LicenseInfo.setLicenseKey('4a83cf3032e8518adfbe8694d092262dTz00ODUxMyxFPTE2OTEyMjE0ODgxODUsUz1wcm8sTE09c3Vic2NyaXB0aW9uLEtWPTI=');
+LicenseInfo.setLicenseKey(process.env.REACT_APP_DATA_GRID_PRO_KEY!);
```

Comment:
The call to the license key should be done once per app:

Copied from earlier comment:

> According to the [Licensing page of MUI](https://mui.com/x/introduction/licensing/#where-to-install-the-key), the key should be exposed in the specific project using the DataGridPro component, and only once in the application, so it shouldn't be called whenever we call the wrapper component.
>
> I suggest deleting this from the component in the Design system, and instead use it in Studio directly, and in storybook so it's only called once per project.